app/src/main/java/woowacourse/shopping/ui/cart/adapter/CartListAdapter.kt

```diff
@@ -15,11 +15,8 @@ class CartListAdapter(
 ) : RecyclerView.Adapter<CartListAdapter.CartListViewHolder>() {
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): CartListViewHolder {
-        val view = LayoutInflater.from(parent.context).inflate(
-            R.layout.item_cart,
-            parent,
-            false,
-        )
+        val view =
+            LayoutInflater.from(parent.context).inflate(R.layout.item_cart, parent, false)
```

Comment:
코드를 깔끔하게 하는 것을 좋아하시는 것 같아 제안을 드립니다.
```suggestion
        val view = LayoutInflater.from(parent.context)
            .inflate(R.layout.item_cart, parent, false)
```
아마 예전 미션에 있던 내용일 거에요.
> 한 줄에는 하나의 점(.)만 사용한다.